README.rst

```diff
@@ -1016,11 +1017,11 @@ Integration tests
 ~~~~~~~~~~~~~~~~~
 
 Integration tests are expected to live in the ``$(INTEGRATIONTEST)`` directory
-(``test`` by default), and it is expected that each subdirectory there is
-a separate test program, with a ``main.d`` file as the entry point. So the
-typical layout for the ``$(INTEGRATIONTEST)/`` directory is::
+(``integrationtest`` by default), and it is expected that each subdirectory
```

Comment:
I think there is an `in` missing in this sentence. It should be `, and it is expected that in each subdirectory there is a ...`.
Anyway this suggestion might not be related to the current changes.

Reply:
Fixed anyway.